src/main/webapp/gameHelper.js

```diff
@@ -296,14 +296,24 @@ function displayEquippedAccessory(object, type) {
       accessoryContainer = document.getElementById(GLASSES_IMAGE_ID);
       break;
   }
-  accessoryContainer.style.zIndex = 3;
-  accessoryContainer.style.position = "absolute";
-  accessoryContainer.style.height = intToDimension(object.height);
-  accessoryContainer.style.width = intToDimension(object.width);
-  accessoryContainer.style.left = intToDimension(object.xPos);
-  accessoryContainer.style.top = intToDimension(object.yPos);
-  accessoryContainer.src = object.imageFilePath;
-  accessoryContainer.name = object.id;
+
+  if (accessoryContainer.name === object.id || object === NONE_EQUIPPED_STRING) {
```

Comment:
Should the order of these checks be reversed? if object == NON_EQUIPPED_STRING won't object.id throw a null pointer when we try to access it first?